src/pl/senla/hotel/ui/ie/ImportExportMenuItem0.java

```diff
@@ -0,0 +1,17 @@
+package pl.senla.hotel.ui.ie;
+
+import pl.senla.hotel.ui.MenuItem;
+
+public class ImportExportMenuItem0 implements MenuItem {
```

Comment:
use readable class names, if you will have 100 menu items you will need to open them one by one to find correct ones, so call it like QuitToMainMenuItem or something like that

Reply:
done